CondFormats/GEMObjects/interface/GEMeMap.h

```diff
@@ -56,18 +56,18 @@ class GEMeMap {
 
 public:
   // size of ID bits
-  static const int vfatVerV3_ = 3;        // VFAT v3
-  static const int vfatTypeV3_ = 11;      // VFAT v3
-  static const int chipIdBits_ = 12;      // ID size from VFat
-  static const int chipIdMask_ = 0xfff;   // chipId mask for 12 bits
-  static const int gebIdBits_ = 5;        // ID size from GEB
-  static const int maxGEBs_ = 32;         // 5 bits for GEB id
-  static const int maxAMCs_ = 16;         // 4 bits for AMC no.
-  static const int maxVFatGE0_ = 12;      // vFat per eta partition, not known yet for ME0
-  static const int maxVFatGE11_ = 3;      // vFat per eta partition in GE11
-  static const int maxVFatGE21_ = 6;      // vFat per eta partition in GE21
-  static const int maxChan_ = 128;        // channels per vFat
-  static const int amcBX_ = 25;           // amc BX to get strip bx
-  static const int maxEtaPartition_ = 8;  // etaPartitions in chamber
+  static const int vfatVerV3_ = 3;         // VFAT v3
+  static const int vfatTypeV3_ = 11;       // VFAT v3
+  static const int chipIdBits_ = 12;       // ID size from VFat
+  static const int chipIdMask_ = 0xfff;    // chipId mask for 12 bits
+  static const int gebIdBits_ = 5;         // ID size from GEB
+  static const int maxGEBs_ = 32;          // 5 bits for GEB id
+  static const int maxAMCs_ = 16;          // 4 bits for AMC no.
+  static const int maxVFatGE0_ = 12;       // vFat per eta partition, not known yet for ME0
+  static const int maxVFatGE11_ = 3;       // vFat per eta partition in GE11
+  static const int maxVFatGE21_ = 6;       // vFat per eta partition in GE21
+  static const int maxChan_ = 128;         // channels per vFat
+  static const int amcBX_ = 25;            // amc BX to get strip bx
+  static const int maxEtaPartition_ = 16;  // etaPartitions in chamber
```

Comment:
Is this a bug fix? (it coincides now with  ```maxRollId = 16``` in DataFormats/MuonDetId/interface/GEMDetId.h)

Reply:
yes. I can remove this and use the consts from GEMDetId if you prefer.

Reply:
Yeah: would that be possible, it will be certainly preferred (and it can also prevent possible further mistakes)

Reply:
Done. 
Currently, maxEtaPartition_ is also used in DQM/GEM/plugins/GEMDQMStatusDigi.cc, so I couldn't remove from GEMeMap.h.
I will get the GEMDQMStatusDigi updated to use the gem parameters from the geometry, then remove this const from GEMeMap.h.

Reply:
Would it be a huge effort updating the GEMDQMStatusDigi.cc to take parameters from the geometry? If not, you can implement everything already in this PR and have it done all at once

Reply:
done!